app/controllers/users_controller.rb

```diff
@@ -17,7 +17,11 @@ def create
   end
 
   def show
+    if current_user == nil
+      redirect_to '/404'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.